View/StrategyCtrView.py

```diff
@@ -247,18 +326,37 @@ def refresh_strat(self, strats):
         else:
             self.selectStrat.addItem('Aucune Stratégie disponible')
 
+    def send_quick_strat1(self):
+        if len(self.strat_default) > 0:
+            self.parent.model_dataout.send_strategy(self.strat_default[0], self.parent.get_team_color())
+
+    def send_quick_strat2(self):
+        if len(self.strat_default) > 1:
+            self.parent.model_dataout.send_strategy(self.strat_default[1], self.parent.get_team_color())
+
+    def send_quick_tactic1(self):
+        if len(self.tactic_default) > 0:
+            self.send_tactic(self.tactic_default[0])
+
+    def send_quick_tactic2(self):
+        if len(self.tactic_default) > 1:
+            self.send_tactic(self.tactic_default[1])
+
     def send_strat(self):
         strat = str(self.selectStrat.currentText())
         if not strat == 'Aucune Stratégie disponible':
             self.parent.model_dataout.send_strategy(strat, self.parent.get_team_color())
 
-    def send_tactic(self):
-        id_bot = int(self.selectRobot.currentText())
+    def send_apply_tactic(self):
         tactic = str(self.selectTactic.currentText())
+        if not tactic == 'Aucune Tactique disponible':
```

Comment:
I'm not sure what is the operation priority here. Can we make this explicit? Also, a comparison with a hardcoded string like that will be hard to maintain.

Reply:
You know `!=` cost a lot of money